src/tests/helpers.js

```diff
@@ -678,83 +684,22 @@ export const schemaInputWithStatement = JSONSchemaBuilder()
         },
       },
     },
-  })
-  .addInput({
-    a_or_b: {
-      title: 'A dropdown',
-      description: 'Some options to chose from',
-      items: {
-        enum: ['A', 'B'],
-      },
+    role: {
+      title: 'Role',
```

Comment:
scout: I've changed the field from select to text because:
- it was using a deprecated way of options.
- the select field was irrelevant either way for this demo.
- makes the json schema smaller, so it's easier to understand [the demo](https://github.com/remoteoss/json-schema-form/assets/14869087/e3c610a1-c21c-40d7-b7b1-f1dc7e431e7b).